mesa/experimental/devs/simulator.py

```diff
@@ -84,6 +109,26 @@ def run_until(self, end_time: int | float) -> None:
                 self._schedule_event(event)  # reschedule event
                 break
 
+    def step(self):
```

Comment:
Could you explain why this step method is added?

Reply:
It allows you do do the next event only which was great for debugging. Open for better names.

Reply:
Maybe we can generalize it to `run_events(n=1)`, with `n` being allowed any integer (and 1 by default)?

Reply:
For now, I prefer to keep it at next event only. It is one of the most powerful things to have when debugging (even from a UI) a model that uses event scheduling.